plugin/trino-hive/src/main/java/io/trino/plugin/hive/coercions/TimestampCoercer.java

```diff
@@ -46,6 +49,9 @@ public final class TimestampCoercer
             .toFormatter()
             .withChronology(IsoChronology.INSTANCE);
 
+    // Before 1900, Java Time and Joda Time are not consistent with java.sql.Date and java.util.Calendar
+    private static final long START_OF_MODERN_ERA_SECONDS = java.time.LocalDate.of(1900, 1, 1).toEpochDay() * SECONDS_PER_DAY;
```

Comment:
Per `io.trino.plugin.deltalake.transactionlog.TransactionLogParser#START_OF_MODERN_ERA` should we add a one day margin since January 1, 1900 UTC is still December 31, 1899 in other zones ?

Reply:
For now - this is specific to timestamp without any timezone - but we could add one day margin.

Reply:
If this is going to get used for timestamp with timezone as well in the future, then I guess we can add the margin now. If not, feel free to skip.

Reply:
For now we won't be having a timestamp with timezone coercion and even in future it would be a in a dedicated class - so we could be having a local variable for avoiding confusion.